content/applications/finance/fiscal_localizations/united_states.rst

```diff
@@ -0,0 +1,746 @@
+    the US, Semi-monthly (24 payments a year) or bi-weekly (26 payments a year) are the most common.
+  - :guilabel:`Wage, Yearly, and Monthly cost`: used to show the total cost of an employee. It is
+    recommended to populate the :guilabel:`Yearly` wage first, as it auto-populates the other
+    fields.
+  - :guilabel:`Pre-tax benefits`: populate this section according to the employee's selections.
+    Pre-tax benefits decrease the gross wage, which lowers the base amount that is taxed. These are
+    displayed at the beginning of the payslip.
+  - :guilabel:`Post-tax benefits`: these benefit are deductions made *after* taxes are calculated.
+    These appear towards the end of the payslip before the net amount is displayed.
+
+.. seealso::
+   :doc:`Employees documentation <../../hr/employees/new_employee>`
+
+.. _l10n_us/adp:
+
+Export work entries to ADP
+--------------------------
+
+Requirements
+~~~~~~~~~~~~
+
+In order to create a report that can be uploaded to ADP, there are some configurations that must be
+done.
+
+First, ensure the *United States - Payroll - Export to ADP* (`l10n_us_hr_payroll_adp`) module is
+:ref:`installed <general/install>`.
+
+Then, the company **must** have an *ADP Code* entered for the company settings. To do so, navigate
+to :menuselection:`Payroll app --> Configuration --> Settings`. Enter the :guilabel:`ADP Code` in
+the :guilabel:`US Localization` section.
+
+Next, work entry types **must** have the correct ADP code listed in the *External Code* field for
+each work entry type that is being referenced.
+
+Lastly, every employee **must** have an *ADP Code* entered on their employee form. To do so,
+navigate to :menuselection:`Employees app`, select an employee record, and open the :guilabel:`HR
+Settings` tab. Enter the :guilabel:`ADP Code` in the :guilabel:`ADP Information` section.
+
+The :guilabel:`ADP Code` code is how ADP identifies that particular employee, and is typically a
+six-digit number.
+
+.. seealso::
+   - :ref:`payroll/new-work-entry`
+   - :doc:`../../hr/employees/new_employee`
+
+Export data
+~~~~~~~~~~~
+
+Once :doc:`work entries <../../hr/payroll/work_entries>` have been verified, the information can be
+exported into a CSV file, which can then be uploaded into ADP.
+
+To export the data, navigate to :menuselection:`Payroll app --> Reporting --> United States: ADP
+Export`, then click :guilabel:`New`. Next, enter the :guilabel:`Start Date` and :guilabel:`End Date`
+for the work entries using the calendar pop-over.
+
+Then, enter a :guilabel:`Batch ID` in the corresponding field. The recommendation for this field is
+to enter the date in a `YY-MM-DD` format, followed by any other characters to distinguish that
+specific batch, such as a department name, or any other defining characteristics for the batch.
+
+Enter a :guilabel:`Batch Description` in the corresponding field. This should be short and
+descriptive, but distinct from the :guilabel:`Batch Name`.
```

Comment:
Screenshot would be helpful here too, to provide a visual model for people to reference.